ethers-etherscan/src/lib.rs

```diff
@@ -247,17 +247,17 @@ impl Client {
 
     /// Return the URL for the given address
     pub fn address_url(&self, address: Address) -> String {
-        format!("{}address/{}", self.etherscan_url, address)
+        format!("{}address/{:?}", self.etherscan_url, address)
```

Comment:
Incredible how often the formatting of the address had to be fixed -.-

Reply:
yeh, I dunno whose idea it was to make `Display` snip the middle 😬